README.md

```diff
@@ -1,24 +1,54 @@
 # Lazyman.bundle
 Lazyman Plex Channel to access Lazyman streams along with recaps and extended highlights.
 
-## Streaming and transcoding
+
+## Streaming and Transcoding
 As far as I gather, the Plex Media Server where the plug-in is installed need to be powerful enough to transcode the HLS (HTTP Live Streaming) streams.
 It might work with a transcoding client such as Plex Media Player or OpenPHT on a computer, not verified.
 Installing the plug-in on a weaker NAS and using a limited client will not work.
 
-### Recaps and highlights
-Recaps and highlights are not part of the Lazyman solution but is offered in the same NHL API and are added for convenience. These are MP4 files and not streams so they should generally work better on most devices. They will also work (or not work) without `/etc/hosts` changes.
+### Recaps and Highlights
+Recaps and highlights are not part of the Lazyman solution but is offered in the same NHL API and are added for convenience. These are MP4 files and not streams so they should generally work better on most devices. They will also work (or not work) without changes to your `hosts` file.
+
+
+## Prerequisites
+### Modify hosts file
+ * Modify your `hosts` file to work with Lazyman on the machine running Plex Media Server:
+ 1. From a command prompt or terminal window, type `ping nhl.freegamez.gq` and note the IP address supplied in the output.
+ 2. Locate the hosts file for your OS.
+    * *The location of this file can vary depending on OS so if you don't know it's best to google for something like "**hosts file location [your OS/NAS here]**".*
+ 3. Modify the hosts file to include the following two lines:
+ ```
+ x.x.x.x mf.svc.nhl.com
+ x.x.x.x mlb-ws-mf.media.mlb.com
+ ```
+ *Where x.x.x.x is the IP address noted from step 1.*
+
 
 ## Installation
-1. [Setup the /etc/hosts file with the Lazyman configuration](https://www.reddit.com/r/nhl_games/comments/5ur6yp/psa_if_the_app_didnt_modify_your_hosts_file/) on the machine running Plex Media Server
-2. [Find your Plex Media Server Plug-in directory](https://support.plex.tv/hc/en-us/articles/201106098-How-do-I-find-the-Plug-Ins-folder-)
-3. [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-4. In the Plug-in folder, clone the repo (install the plugin): `git clone https://github.com/nomego/Lazyman.bundle`
+### Unsupported AppStore Method
+*If you already have WebTools/UAS installed, skip to step 2.*
+1. Install the [WebTools](https://github.com/ukdtom/WebTools.bundle/wiki/V2Home) PMS Plug-in.
```

Comment:
Where can I see the listed Plug-ins (without installing the UAS) and who listed my repo?

Reply:
Upon further review, it looks like the UAS crew added @unclenoriega 's fork and not your bundle. My apologies for stirring that up. I can either modify the instructions to use the semi-automatic installer (which is the thing I hint at in the note just below these steps), or I can remove the UAS option altogether. I do not know how you get a bundle added to the UAS list, but I did find [this ongoing thread](https://forums.plex.tv/discussion/200017/support-find-my-plug-in-here-for-support-read-me-first#L) over at the Plex forums. Let me know how you'd like to proceed.

Reply:
Someone on reddit asked me to add my fork to UAS, and then I switched it to point to your repo after our conversation. I can ask them to remove it if you don't want it in there.

Reply:
Just curious! If people use it to install the channel then we should keep it there. Thanks!